api/handlers_test.go

```diff
@@ -29,6 +31,7 @@ func GetAPIWithMocks(mockedDataStore store.Storer) *RecipeAPI {
 	So(err, ShouldBeNil)
 	cfg.MongoConfig.EnableMongoData = true
 	cfg.MongoConfig.EnableMongoImport = true
+	cfg.MongoConfig.EnableAuthImport = true
```

Comment:
Is it possible to test the response when the feature flag is set to false?

Reply:
When I set the feature flag to false and run the tests, I get a `405 Status - Method Not Allowed`. This is what we expect to happen when the feature flag is turned off as we can only use the endpoint when flag enabled

Reply:
Is it worth adding a test to ensure this behaviour happens when toggled off?